llpc/test/shaderdb/core/OverrideThreadGroupSize16X16X1.spvasm

```diff
@@ -0,0 +1,102 @@
+; BEGIN_SWIZZLETEST
+; RUN: amdllpc -v %gfxip %s --override-threadGroupSizeX=16 --override-threadGroupSizeY=16 --override-threadGroupSizeZ=1 | FileCheck -check-prefix=OVERRIDETHREADGROUPTEST %s
```

Comment:
This line is very long and does not fit on my screen. Can you please break it, something like:
```
; RUN: amdllpc -v %gfxip %s \
; RUN:   --override-threadGroupSizeX=16 --override-threadGroupSizeY=16 --override-threadGroupSizeZ=1 \
; RUN:  |  FileCheck -check-prefix=SHADERTEST %s
```

Reply:
Also, I'd use `CHECK` or `SHADERTEST` over `OVERRIDETHREADGROUPTEST` which is a bit of a mouthful.

Reply:
OK. I would like follow these two suggestions.